# The Construction of Reference Gene and Genome Catalogs of Chinese Liquor
Necessary resources consist of softwares, algorithms and databases had been provided.
|   **REAGENT or RESOURCES**  |      **Version**      |                   **SOURCE**                   |                   **IDENTIFIER**                  |
|:---------------------------:|:---------------------:|:----------------------------------------------:|:-------------------------------------------------:|
| **Software and algorithms** |                       |                                                |                                                   |
|         Trimmomatic         |         v0.38         |  https://doi.org/10.1093/bioinformatics/btu170 |      https://github.com/usadellab/Trimmomatic     |
|           MEGAHIT           |         v1.1.2        |  https://doi.org/10.1093/bioinformatics/btv033 |         https://github.com/voutcn/megahit         |
|            QUAST            |         v5.0.2        |  https://doi.org/10.1093/bioinformatics/btt086 |           https://github.com/ablab/quast          |
|           MetaWRAP          |         v1.2.2        |    https://doi.org/10.1186/s40168-018-0541-1   |         https://github.com/bxlab/metaWRAP         |
|           CheckM            |        v1.0.18        |      https://doi.org/10.1101/gr.186072.114     |         https://github.com/tribe29/checkmk        |
|          Prodigal           |         v2.6.3        |    https://doi.org/10.1186/1471-2105-11-119    |        https://github.com/hyattpd/Prodigal        |
|            Prokka           |        v1.14.5        |  https://doi.org/10.1093/bioinformatics/btu153 |         https://github.com/tseemann/prokka        |
|           MMseqs2           |         v4.8.1        |  https://doi.org/10.1093/bioinformatics/btq003 |       https://github.com/soedinglab/MMseqs2       |
|          KofamScan          |         v1.3.0        |  https://doi.org/10.1093/bioinformatics/btz859 | https://github.com/rotheconrad/KEGGDecoder-binder |
|           DIAMOND           |      v2.0.14.152      |       https://doi.org/10.1038/nmeth.3176       |     https://github.com/python-diamond/Diamond     |
|           DeepARG           |         v1.0.2        |    https://doi.org/10.1186/s40168-018-0401-z   |         https://github.com/Deeparg/Deeparg        |
|        eggNOG-mapper        |         v2.1.6        |     https://doi.org/10.1093/molbev/msab293     |     https://github.com/eggnogdb/eggnog-mapper     |
|          antiSMASH          |         v6.0.0        |       https://doi.org/10.1093/nar/gkab335      |       https://github.com/antismash/antismash      |
|          BiG-SCAPE          |         v1.1.5        |    https://doi.org/10.1038/s41589-019-0400-9   |     https://github.com/medema-group/BiG-SCAPE     |
|             dRep            |         v3.4.0        |     https://doi.org/10.1038/ismej.2017.126     |           https://github.com/MrOlm/drep           |
|           GTDB-tk           |         v2.1.1        | https://doi.org/10.1093/bioinformatics/btac672 |       https://gtdb.ecogenomic.org/downloads       |
|          Rosetta2           |          v2.0         |     https://doi.org/10.1006/jmbi.1997.0959     |      https://github.com/h33p/docker-rosetta2      |
|        **Databases**        | **Date for download** |                   **SOURCE**                   |                   **IDENTIFIER**                  |
|            eggNOG           |     v5.0; 2022_12     |       https://doi.org/10.1093/nar/gky1085      |       http://eggnog5.embl.de/#/app/downloads      |
|             KEGG            |        2022_12        |       https://doi.org/10.1093/nar/gkaa970      |    http://kobas.cbi.pku.edu.cn/kobas3/download/   |
|             COG             |        2022_12        |       http://oi.org/10.1093/nar/gkaa1018       |          http://www.ncbi.nlm.nih.gov/COG/         |
|              GO             |        2022_12        |        http://oi.org/10.1093/nar/gky1055       |              http://geneontology.org/             |
|            CAZyDB           |        2022_12        |       https://doi.org/10.1093/nar/gkn663       |                http://www.cazy.org/               |
|              EC             |        2022_12        |       http://oi.org/10.1093/nar/28.1.304       |        https://enzyme.expasy.org/index.html       |
|             CARD            |        2022_12        |       https://doi.org/10.1093/nar/gkz935       |         https://card.mcmaster.ca/download         |
|             VFDB            |        2022_12        |       https://doi.org/10.1093/nar/gki008       |       http://www.mgc.ac.cn/VFs/download.htm       |
|             GTDB            |        2022_12        | https://doi.org/10.1093/bioinformatics/btac672 |       https://gtdb.ecogenomic.org/downloads       |
|         Swiss-sport         |        2022_12        |      https://doi.org/10.1093/nar/gkac1052      |              https://www.uniprot.org/             |
|            TrEMBL           |        2022_12        |      https://doi.org/10.1093/nar/gkac1052      |              https://www.uniprot.org/             |
|           UniRef50          |        2022_12        |  https://doi.org/10.1093/bioinformatics/btm098 |              https://www.uniprot.org/             |
|             Pfam            |        2022_12        |               https://doi.org/10.1093/nar/gkaa913              |            http://pfam-legacy.xfam.org/           |

**Data availability**

The raw metagenomic data could be assessed at the GSA database (https://ngdc.cncb.ac.cn/gsub/) with ProjectID: PRJCA018633 for fermented grain samples and PRJCA018634 for starter samples.
