# The Construction of Reference Gene and Genome Catalogs of Chinese Liquor
Necessary resources consist of softwares, algorithms and databases had been provided.

| **reagent or resources**    | **version**                | **identifier**                                        | **source**                                         |
|:-----------------------:|:----------------------:|:-------------------------------------------------:|:----------------------------------------------:|
| **software and algorithms** |                        |                                                   |                                                |
| trimmomatic             | v0.38                  | https://github.com/usadellab/trimmomatic          | https://doi.org/10.1093/bioinformatics/btu170  |
| metaphlan2              | v2.6.0                 | https://github.com/biobakery/metaphlan2           | https://doi.org/10.1038/nmeth.3589             |
| megahit                 | v1.1.2                 | https://github.com/voutcn/megahit                 | https://doi.org/10.1093/bioinformatics/btv033  |
| fastnc2                 | v2                     | https://github.com/wqssf102/fastnc2               | https://doi.org/10.1073/pnas.202532111         |
| quast                   | v5.0.2                 | https://github.com/ablab/quast                    | https://doi.org/10.1093/bioinformatics/btt086  |
| metawrap                | v1.2.2                 | https://github.com/bxlab/metawrap                 | https://doi.org/10.1186/s40168-018-0541-1      |
| checkm                  | v1.0.18                | https://github.com/tribe29/checkmk                | https://doi.org/10.1101/gr.186072.114          |
| prodigal                | v2.6.3                 | https://github.com/hyattpd/prodigal               | https://doi.org/10.1186/1471-2105-11-119       |
| prokka                  | v1.14.5                | https://github.com/tseemann/prokka                | https://doi.org/10.1093/bioinformatics/btu153  |
| mmseqs2                 | v4.8.1                 | https://github.com/soedinglab/mmseqs2             | https://doi.org/10.1093/bioinformatics/btq003  |
| kofamscan               | v1.3.0                 | https://github.com/rotheconrad/keggdecoder-binder | https://doi.org/10.1093/bioinformatics/btz859  |
| diamond                 | v2.0.14.152            | https://github.com/python-diamond/diamond         | https://doi.org/10.1038/nmeth.3176             |
| deeparg                 | v1.0.2                 | https://github.com/deeparg/deeparg                | https://doi.org/10.1186/s40168-018-0401-z      |
| eggnog-mapper           | v2.1.6                 | https://github.com/eggnogdb/eggnog-mapper         | https://doi.org/10.1093/molbev/msab293         |
| antismash               | v6.0.0                 | https://github.com/antismash/antismash            | https://doi.org/10.1093/nar/gkab335            |
| big-scape               | v1.1.5                 | https://github.com/medema-group/big-scape         | https://doi.org/10.1038/s41589-019-0400-9      |
| drep                    | v3.4.0                 | https://github.com/mrolm/drep                     | https://doi.org/10.1038/ismej.2017.126         |
| gtdb-tk                 | v2.1.1                 | https://gtdb.ecogenomic.org/downloads             | https://doi.org/10.1093/bioinformatics/btac672 |
| rosetta2                | v2.0                   | https://github.com/h33p/docker-rosetta2           | https://doi.org/10.1006/jmbi.1997.0959         |
| **databases**               | **date for download**      | **identifier**                                        | **source**                                         |
| eggnog                  | v5.0; 2022_12          | http://eggnog5.embl.de/#/app/downloads            | https://doi.org/10.1093/nar/gky1085            |
| kegg                    | 2022_12                | http://kobas.cbi.pku.edu.cn/kobas3/download/      | https://doi.org/10.1093/nar/gkaa970            |
| cog                     | 2022_12                | http://www.ncbi.nlm.nih.gov/cog/                  | http://oi.org/10.1093/nar/gkaa1018             |
| go                      | 2022_12                | http://geneontology.org/                          | http://oi.org/10.1093/nar/gky1055              |
| cazydb                  | 2022_12                | http://www.cazy.org/                              | https://doi.org/10.1093/nar/gkn663             |
| ec                      | 2022_12                | https://enzyme.expasy.org/index.html              | http://oi.org/10.1093/nar/28.1.304             |
| card                    | 2022_12                | https://card.mcmaster.ca/download                 | https://doi.org/10.1093/nar/gkz935             |
| gtdb                    | release207_v2; 2022_12 | https://gtdb.ecogenomic.org/downloads             | https://doi.org/10.1093/bioinformatics/btac672 |
| swiss-sport             | 2022_12                | https://www.uniprot.org/                          | https://doi.org/10.1093/nar/gkac1052           |
| trembl                  | 2022_12                | https://www.uniprot.org/                          | https://doi.org/10.1093/nar/gkac1052           |
| uniref50                | 2022_12                | https://www.uniprot.org/                          | https://doi.org/10.1093/bioinformatics/btm098  |
| pfam                    | 2022_12                | http://pfam-legacy.xfam.org/                      | https://doi.org/10.1093/nar/gkaa913            |


**Data availability**

The raw metagenomic data could be assessed at the GSA database (https://ngdc.cncb.ac.cn/gsub/) with ProjectID: PRJCA018633 for fermented grain samples and PRJCA018634 for starter samples.
