# The Construction of Reference Gene and Genome Catalogs of Chinese Liquor
Necessary resources consist of softwares, algorithms and databases had been provided.
| **Key Resource Table**  | ****                   | ****                                              | ****                                           |
|:-----------------------:|:----------------------:|:-------------------------------------------------:|:----------------------------------------------:|
|                         |                        |                                                   |                                                |
|                         |                        |                                                   |                                                |
| REAGENT or RESOURCES    | Version                | IDENTIFIER                                        | SOURCE                                         |
| Software and algorithms |                        |                                                   |                                                |
| Trimmomatic             | v0.38                  | https://github.com/usadellab/Trimmomatic          | https://doi.org/10.1093/bioinformatics/btu170  |
| MetaPhlAn2              | v2.6.0                 | https://github.com/biobakery/MetaPhlAn2           | https://doi.org/10.1038/nmeth.3589             |
| MEGAHIT                 | v1.1.2                 | https://github.com/voutcn/megahit                 | https://doi.org/10.1093/bioinformatics/btv033  |
| FastNC2                 | v2                     | https://github.com/wqssf102/fastnc2               | https://doi.org/10.1073/pnas.202532111         |
| QUAST                   | v5.0.2                 | https://github.com/ablab/quast                    | https://doi.org/10.1093/bioinformatics/btt086  |
| MetaWRAP                | v1.2.2                 | https://github.com/bxlab/metaWRAP                 | https://doi.org/10.1186/s40168-018-0541-1      |
| CheckM                  | v1.0.18                | https://github.com/tribe29/checkmk                | https://doi.org/10.1101/gr.186072.114          |
| Prodigal                | v2.6.3                 | https://github.com/hyattpd/Prodigal               | https://doi.org/10.1186/1471-2105-11-119       |
| Prokka                  | v1.14.5                | https://github.com/tseemann/prokka                | https://doi.org/10.1093/bioinformatics/btu153  |
| MMseqs2                 | v4.8.1                 | https://github.com/soedinglab/MMseqs2             | https://doi.org/10.1093/bioinformatics/btq003  |
| KofamScan               | v1.3.0                 | https://github.com/rotheconrad/KEGGDecoder-binder | https://doi.org/10.1093/bioinformatics/btz859  |
| DIAMOND                 | v2.0.14.152            | https://github.com/python-diamond/Diamond         | https://doi.org/10.1038/nmeth.3176             |
| DeepARG                 | v1.0.2                 | https://github.com/Deeparg/Deeparg                | https://doi.org/10.1186/s40168-018-0401-z      |
| eggNOG-mapper           | v2.1.6                 | https://github.com/eggnogdb/eggnog-mapper         | https://doi.org/10.1093/molbev/msab293         |
| antiSMASH               | v6.0.0                 | https://github.com/antismash/antismash            | https://doi.org/10.1093/nar/gkab335            |
| BiG-SCAPE               | v1.1.5                 | https://github.com/medema-group/BiG-SCAPE         | https://doi.org/10.1038/s41589-019-0400-9      |
| dRep                    | v3.4.0                 | https://github.com/MrOlm/drep                     | https://doi.org/10.1038/ismej.2017.126         |
| GTDB-tk                 | v2.1.1                 | https://gtdb.ecogenomic.org/downloads             | https://doi.org/10.1093/bioinformatics/btac672 |
| Rosetta2                | v2.0                   | https://github.com/h33p/docker-rosetta2           | https://doi.org/10.1006/jmbi.1997.0959         |
| Databases               | Date for download      | IDENTIFIER                                        | SOURCE                                         |
| eggNOG                  | v5.0; 2022_12          | http://eggnog5.embl.de/#/app/downloads            | https://doi.org/10.1093/nar/gky1085            |
| KEGG                    | 2022_12                | http://kobas.cbi.pku.edu.cn/kobas3/download/      | https://doi.org/10.1093/nar/gkaa970            |
| COG                     | 2022_12                | http://www.ncbi.nlm.nih.gov/COG/                  | http://oi.org/10.1093/nar/gkaa1018             |
| GO                      | 2022_12                | http://geneontology.org/                          | http://oi.org/10.1093/nar/gky1055              |
| CAZyDB                  | 2022_12                | http://www.cazy.org/                              | https://doi.org/10.1093/nar/gkn663             |
| EC                      | 2022_12                | https://enzyme.expasy.org/index.html              | http://oi.org/10.1093/nar/28.1.304             |
| CARD                    | 2022_12                | https://card.mcmaster.ca/download                 | https://doi.org/10.1093/nar/gkz935             |
| GTDB                    | release207_v2; 2022_12 | https://gtdb.ecogenomic.org/downloads             | https://doi.org/10.1093/bioinformatics/btac672 |
| Swiss-sport             | 2022_12                | https://www.uniprot.org/                          | https://doi.org/10.1093/nar/gkac1052           |
| TrEMBL                  | 2022_12                | https://www.uniprot.org/                          | https://doi.org/10.1093/nar/gkac1052           |
| UniRef50                | 2022_12                | https://www.uniprot.org/                          | https://doi.org/10.1093/bioinformatics/btm098  |
| Pfam                    | 2022_12                | http://pfam-legacy.xfam.org/                      | https://doi.org/10.1093/nar/gkaa913            |
|                         |                        |                                                   |                                                |
|                         |                        |                                                   |                                                |
|                         |                        |                                                   |                                                |
|                         |                        |                                                   |                                                |



**Data availability**

The raw metagenomic data could be assessed at the GSA database (https://ngdc.cncb.ac.cn/gsub/) with ProjectID: PRJCA018633 for fermented grain samples and PRJCA018634 for starter samples.
